src/vl-map-delete-action.js

```diff
@@ -37,10 +37,32 @@ export class VlMapDeleteAction extends VlMapLayerAction {
     }
   }
 
+  /**
+   * Zet de functie die wordt uitgevoerd na het uitvoeren van de verwijder actie
+   *
+   * @param {Function} callback functie met volgende argumenten:
+   *                            - {[ol.Feature]} de te verwijderen features
+   *                            - {Function} resolve callback met ol.Feature als argument die verwijderd wordt
+   *                            - {Function} reject callback zonder argument waarbij de highlight verwijderd wordt
+   */
+  onDelete(callback) {
+    this.__callback = callback;
```

Comment:
__callback is misschien een te generieke naam voor een on-delete callback?

Reply:
Ik zie dat __callback in iedere actie vervat zit. Aan de ene kant is het natuurlijk de callback voor de actie, aan de andere kant wil je misschien bij bepaalde acties ook onderdelen van de actie callbacken (bv. bij elke lijn van een polygoon). Nuja, YAGNI: het is toch maar een private var.